### PR TITLE
Introduce Azure documentation

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: plugins/foreman_azure/1.x/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    $ gem 'foreman_azure'
+
+Then run `bundle install` from the same directory
+
+# 3. Configuration
+
+Same as any other compute resource, provide your credentials:
+
+![Azure credentials](http://i.imgur.com/4uOVa2W.png)
```
Comment (domcleal) [inline plugins/foreman_azure/1.x/index.md]: These images should be in the repo too, or at the very least on an HTTPS site.

File: plugins/foreman_azure/1.x/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+layout: plugin
+title: Foreman Azure 1.x Manual
+version: 1.x
+---
+
+# 1. Basics
+
+Foreman Azure allows you to provision and manage your hosts in Azure. This is convenient when you want your users to have restricted access to your Azure account (such that only they can create hosts), to monitor and configure them automatically (via Ansible, Puppet, etc..), or also to automate its creation via hostgroups.
```
Comment (domcleal) [inline plugins/foreman_azure/1.x/index.md]: Last word: "host groups"

File: plugins/foreman_azure/1.x/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+This plugin 1.x series should be compatible with Foreman 1.11, 1.12 and 1.13. Future versions of Foreman might or might not be comptible. 2.x will be released when the plugin is found to not be compatible with any of these 3 versions or drops compatibility of any of them.
```
Comment (domcleal) [inline plugins/foreman_azure/1.x/index.md]: Spelling: "not be compatible"

File: plugins/foreman_azure/1.x/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+# 2. Installation
+
+See [the Foreman manual](https://theforeman.org/plugins/#2.2Packageinstallation).
```
Comment (domcleal) [inline plugins/foreman_azure/1.x/index.md]: Use a relative URL here.

File: plugins/foreman_azure/1.x/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+# 5. Help
+
+Please follow our [standard procedures and contacts]({{site.baseurl}}support.html).
+
+## 5.1 Troubleshooting
+
+Errors regarding the provisioning will show up on /var/log/foreman/production.log of your Foreman host.
+
+If you find a bug, please file it in
+[Redmine](http://projects.theforeman.org/projects/foreman_azure/issues/new).
```
Comment (domcleal) [inline plugins/foreman_azure/1.x/index.md]: Broken link, project ID is "azure"

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by dLobatog** [comment]
Updated, thanks @domcleal !

**Review by domcleal — APPROVED** [review]

**Comment by domcleal** [comment]
Merged, thanks @dLobatog.

**Pull request merged**